en/docs/devops-and-ci-cd/manage-configuration-groups.md

```diff
@@ -0,0 +1,112 @@
+
+# Manage Configuration Groups
+
+**Configuration groups** are collections of key-value pairs that can be linked to any component within the organization, enabling centralized configuration management.
+
+Configuration values can be maintained for a set of environments. During deployment, a configuration group can be linked to inject its keys as environment variables or file mounts. Choreo manages the secrets and configurations, ensuring they are applied correctly during deployment and promotion.
+
+!!!important
+    All the values provided for configuration groups are encrypted and stored in the environment specific key vaults.
```

Comment:
Suggestion:
All configuration group values are encrypted and stored in environment-specific key vaults.